EventKit/eventkit-cloud
# fixes issue with styles, caching, and skips empty requests

# Description of Improvement

Fixes issue with the cachable helper to prevent repeated requests.
Fixes issue with missing attribute from styles.  Catches the error and prints the style sheet info. 
Skips additional arcgis feature requests if they have no data. 

## How to test

Create an export using an arcgis feature service.  Ensure that everything exports as expected. 

## Quality Assurance

The following items have been updated:


## Screenshots



:smile:
